src/main/java/task/Task.java

```diff
@@ -0,0 +1,85 @@
+package task;
+/** This class represents a Task in which the User can store and list them in Sir Duke */
+public class Task {
+    /** Description of Task */
+    protected String description;
+    /** is true when task isDone */
+    protected boolean isDone;
```

Comment:
I like how you followed naming conventions and made boolean variables sound more obvious as boolean variables.